README.md

```diff
@@ -36,9 +36,67 @@ Try out this [JS Bin](https://jsbin.com/govola/10/edit?js,output).
 
 See a real world example: [cycle autocomplete](https://github.com/cyclejs-community/redux-cycles/blob/master/example/cycle/index.js).
 
-## What is this?
+## Why?
 
-Here's how Async is done using [redux-observable](https://github.com/redux-observable/redux-observable). The problem is that we still have side-effects in our epics (`ajax.getJSON`). This means that we're still writing imperative code:
+There already are several side-effects solutions in the Redux ecosystem:
+
+* [redux-thunk](https://github.com/gaearon/redux-thunk)
+* [redux-saga](https://github.com/redux-saga/redux-saga)
+* [redux-ship](https://clarus.github.io/redux-ship/)
+* [redux-observable](http://redux-observable.js.org)
+
+Why create yet another one?
+
+The intention with redux-cycles was not to worsen the "JavaScript fatigue".
+Rather it provides a solution that solves several problems attributable to the currently available libraries.
+
+* **Respond to actions as they happen, from the side.**
+
+  Redux-thunk forces you to put your logic directly into the action creator.
+  This means that all the logic caused by a particular action is located in one place... which doesn't do the readability a favor.
+  It also means cross-cutting concerns like analytics get spread out across many files and functions.
+
+  Redux-cycles, instead, joins redux-saga and redux-observable in allowing you to respond to actions reactively.
```

Comment:
I tried to avoid the word "reactive" because for a lot of people it means observables but that's not what I meant here. I couldn't find any other word to express that both sagas and observables *react* to actions, instead of being embedded inside action creators